tests/queries/0_stateless/02404_rocksdb_read_only.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, no-parallel
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+ROCKSDB_DIR="/var/lib/clickhouse/store/test_rocksdb_read_only"
+
+$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS test_02404;" 2>&1 | grep -F -q "OK" && echo 'OK' || echo 'FAIL';
```

Comment:
```suggestion
$CLICKHOUSE_CLIENT --query="DROP TABLE IF EXISTS test_02404;" && echo 'OK' || echo 'FAIL';
```
All checks should work in this form

Reply:
then the result of the query will so be printed?